apps/cms/src/migrations/20240616_101649_Add_displayTitle_to_database.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  type MigrateUpArgs,
+  type MigrateDownArgs,
+} from "@payloadcms/db-mongodb";
+import { type PayloadRequest } from "payload/types";
+import { CommitteeMembers } from "../collections/committees/committee-members";
+import { NewsItems } from "../collections/weekly-newsletters/news-items";
+
+export async function up({ payload }: MigrateUpArgs): Promise<void> {
+  const req = {} as PayloadRequest;
+
+  const committeeMembers = await payload.find({
```

Comment:
Does this work without specifying a higher limit than default? I suspect there could be a limit here even with pagination false where it wouldn't apply to all items